pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -132,8 +132,7 @@ func NewManager(cpuPolicyName string, reconcilePeriod time.Duration, machineInfo
 		policy = NewStaticPolicy(topo, numReservedCPUs)
 
 	default:
-		klog.Errorf("[cpumanager] Unknown policy \"%s\", falling back to default policy \"%s\"", cpuPolicyName, PolicyNone)
-		policy = NewNonePolicy()
+		return nil, fmt.Errorf("unknown policy: \"%s\"", cpuPolicyName)
```

Comment:
Just confirming that currently only the test suite calls this function? Is this because this is added as part of the topology work that is still WIP?

Reply:
Yes. Please see: https://github.com/kubernetes/kubernetes/pull/74357/files#r304869037